website/docs/examples/camera_calibration/README.md

```diff
@@ -45,7 +47,10 @@ python3 main.py --service-config service_config.json
 ```
 
 :::info
-By default, the camera address is assumed top be `localhost`.
+By default, the camera address is assumed to be `localhost` (service_config.json - line 4).
```

Comment:
Use the field name rather than the line number (line numbers a re subject to change, field names should stay static)
```suggestion
By default, the camera host is assumed to be `localhost`.
```